dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -543,3 +594,32 @@ describe('Clean up all test data', () => {
   });
 });
 
+describe('Verify Operational Panels Table', () =>{
+  it('Add Dummy Data to Operational Panel and verify Table Column, Pagination and Rows Data ', () => {
+  let p = 0;
+  for (p = 0; p < 12 ; p++) { 
+  createDummyPanel(); }
```

Comment:
Indentation is not proper. The closing `}` should not be in the same line. `let` can also be written inside the loop `for(let p = 0; .....)`